.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04, actuated-arm64-6cpu-8gb]
-        go-version: [1.20.x, 1.21.x]
+        go-version: [1.20.x, 1.21.x, tip]
```

Comment:
So we have to alway test the unreleased go version in the future? If there was something wrong in master branch of go repository, we would get ci failure too.

Reply:
> So we have to alway test the unreleased go version in the future?

No, we don't.

The reason for adding this is simple. The code added is for Go 1.23, and there's no other way to test it, and I don't want to push the code that's not tested.

I can remove the gotip commit now when we saw it's working.

Reply:
Removed; PTAL @lifubang